var/spack/repos/builtin/packages/py-multiqc/package.py

```diff
@@ -34,7 +36,6 @@ class PyMultiqc(PythonPackage):
     depends_on("py-jinja2@2.9:", type=("build", "run"), when="@:1.13")
     depends_on("py-kaleido", type=("build", "run"), when="@1.20:")
     depends_on("py-markdown", type=("build", "run"), when="@1.3:")
-    depends_on("py-matplotlib@2.1.1:", type=("build", "run"), when="@1.13:")
```

Comment:
Do we need to make it `when=":@1.21"` rather than flat out deleting this line?

Reply:
We can delete old versions, of course